applications/tari_app_utilities/Cargo.toml

```diff
@@ -33,3 +33,6 @@ features = ["transactions"]
 
 [build-dependencies]
 tari_common = {  path = "../../common", features = ["build", "static-application-info"] }
+
+[features]
```

Comment:
This shows that there is an incorrect abstraction. This library is meant to house common functionality between the two (and more) apps. If there is code specific to the wallet, it needs to be moved into that app specifically

Reply:
Yeah does need some sorting, I had finished the work with the base node but could not compile it because it depended on the wallet - unfortunately because both the type and trait (From<T>) is foreign to tari_wallet we cannot just move the code. But we should sort this out.